src/components/Request/Pay/Views/Success.view.tsx

```diff
@@ -18,10 +21,32 @@ export const SuccessView = ({ transactionHash, requestLinkData }: _consts.IPaySc
     const explorerUrlAxelarWithTx = 'https://axelarscan.io/gmp/' + transactionHash
 
     useEffect(() => {
-        if (transactionHash) {
+        if (explorerUrlDestChainWithTxHash) {
+            peanut.submitRequestLinkFulfillment({
+                chainId: requestLinkData.chainId,
+                hash: explorerUrlDestChainWithTxHash.transactionId,
```

Comment:
Question: Just so I understand this, does the requestLinkFulfillment only get submitted once the destination chain tx is through? So, if the user closes the page, it would never submit the fulfillment right?

Reply:
Correct. I think the way to go around this is submit the origin transaction to the backend (but not mark as paid) and then with the destination chain tx we can do as today (if the user did not close the page) or in the backend we can check